hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OzoneClientKeyReadWriteOps.java

```diff
@@ -229,15 +236,20 @@ public TaskType decideReadOrWriteTask() {
 
   public String getKeyName() {
     StringBuilder keyNameSb = new StringBuilder();
-    int randomIdxWithinRange = ThreadLocalRandom.current().
-            nextInt(startIndex, startIndex + range);
+    int next;
+    if (linear) {
+      next = nextNumber.getAndUpdate(x -> (x + 1) % (startIndex + range + 1));
```

Comment:
Does this result in a number in the range of `[0, startIndex + range]`, while we actually want a sequential cursor in the range [startIndex, range)?

If that's true, something like the below could work
```
   next = nextNumber.increaseAndGet() % range + startIndex;
```

Reply:
Thanks